portfolio/src/main/webapp/index.html

```diff
@@ -85,6 +71,25 @@ <h2>Comment Section</h2>
               <div id="page-number-container"></div>
           </div>
 
+          <div id="projects-container" class="info-container">
+              <h2>Projects</h2>
+              <ul>
+                  <li><strong>AVMisc</strong></li>
```

Comment:
I'm not going to require this, because I'm not much of an expert on html/css, but I believe it's frowned upon to specify style like \<strong\> in the html, and the preferred method is to use CSS to drive all style.  Then if you decide that you want these elements to have a slightly different style, you can change the CSS in one place, rather than having to edit the html in several different locations.

I might be wrong, though - so let me know if you know/feel differently!

Reply:
I think that makes sense! I'll make the changes.